public/js/components/AtomEdit/AtomEdit.js

```diff
@@ -31,12 +32,21 @@ class AtomEdit extends React.Component {
   }
 
   render () {
-    return (
-      <dev>
-        <p>Editing atom: </p>
-        <button className="btn" type="button" onClick={this.updateAtom}>Update atom</button>
-      </dev>
-    );
+
+    if (!this.props.atom) {
+      return <div>Loading...</div>;
+    }
+
+    const atomType = this.props.atom.atomType.toLowerCase();
```

Comment:
I know this is linked to the data coming back but I'm not a fan. Maybe we could add a note?